spec/controllers/idv/forgot_password_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
```

Comment:
Looks like we don't have any feature spec coverage of this. An idv feature spec for resetting a password and signing back in with the new password to complete IdV seems like a good idea to me.

Reply:
done